### PR TITLE
Youtube page clone completed

### PR DESCRIPTION
**Pull request by voscarmv**
Finished cloning the youtube page and completing the requirements from Microverse.

### CODE REVIEW

**Comment by Anna-Myzukina** [comment]
### Pull Request under the review by Anna Muzykina (TSE)

**Comment by Anna-Myzukina** [comment]
### Hi team !
### Nice job! Project looks! :+1: 

### But your project needs a few changes before it can be approved.

### Page structure

#### Navigation bar
screen

* In the original version the search button is at the end (rightmost part) of the search bar
searchoriginal

* In your version this button is absent
search

* Kindly recommended checking dislike button, now it`s not displaying
dislike


### General
* Comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history
screnn


* Also please read this article [Reasons not to use IDs in CSS](https://dev.to/clairecodes/reasons-not-to-use-ids-in-css-4ni4) I hope it can help you a little.

### Make changes and submit another code review request.

*If you have any questions, let me know on slack (@Anna-Muzykina) and I will be glad to clarify with you.*

*Happy coding 😄*

File: style.css [hunk 1]
```diff
@@ -0,0 +1,397 @@
...
+	padding-left: 5px;
+	padding-right: 4px;
+	width: 22px;
+	display: flex;
+	align-items: center;
+	justify-content: center;
+	text-align: center;
+}
+#plus-sign-icon-image {}
+
```
Comment (Anna-Myzukina) [inline style.css:232]: Do not use empty rulesets CSS, line number 231 better to be deleted

File: style.css [hunk 2]
```diff
@@ -0,0 +1,397 @@
...
+}
+
+.search-button {
+	border: 1px solid black;
+	height: 36px;
+	padding: 10px 10px;
+}
+
+.upload-button {}
+
```
Comment (Anna-Myzukina) [inline style.css:52]: Do not use empty rulesets CSS, line number 51 better to be deleted

**Review by Anna-Myzukina — CHANGES_REQUESTED** [review]
The project needs changes

File: style.css (hunk 1 above)
Comment (dannisonarias) [reply style.css:232]: Thank you very much Anna, we're now working on these issues.

**Pull request merged**